e2e-tests/playwright/e2e/audit-log/catalog.spec.ts

```diff
@@ -0,0 +1,115 @@
+    eventName: string,
+    message: string,
+    method: string,
+    apiPath: string,
+    plugin: string = 'catalog',
+  ) {
+    await LogUtils.validateLogEvent(
+      eventName,
+      message,
+      method,
+      apiPath,
+      baseApiUrl,
+      plugin,
+    );
+  }
+
+  test('Should fetch logs for CatalogEntityFacetFetch event and validate log structure and values', async () => {
+    await validateCatalogLogEvent(
+      'CatalogEntityFacetFetch',
+      'Entity facet fetch attempt',
+      'GET',
+      '/api/catalog/entity-facets',
+    );
+  });
+
+  test('Should fetch logs for CatalogEntityFetchByName event and validate log structure and values', async () => {
+    await uiHelper.selectMuiBox('Kind', 'Component');
+    await uiHelper.clickByDataTestId('user-picker-all');
+    await uiHelper.clickLink('backstage-janus');
+    await validateCatalogLogEvent(
+      'CatalogEntityFetchByName',
+      'Fetch attempt for entity with entityRef component:default/backstage-janus',
+      'GET',
+      '/api/catalog/entities/by-name/component/default/backstage-janus',
+    );
+  });
+
+  test('Should fetch logs for CatalogEntityBatchFetch event and validate log structure and values', async () => {
+    await uiHelper.selectMuiBox('Kind', 'Component');
+    await uiHelper.clickByDataTestId('user-picker-all');
+    await uiHelper.clickLink('backstage-janus');
+    await validateCatalogLogEvent(
+      'CatalogEntityBatchFetch',
+      'Batch entity fetch attempt',
+      'POST',
+      '/api/catalog/entities/by-refs',
+    );
+  });
+
+  test('Should fetch logs for CatalogEntityAncestryFetch event and validate log structure and values', async () => {
+    await uiHelper.selectMuiBox('Kind', 'Component');
+    await uiHelper.clickByDataTestId('user-picker-all');
+    await uiHelper.clickLink('backstage-janus');
+    await validateCatalogLogEvent(
+      'CatalogEntityAncestryFetch',
+      'Fetch attempt for entity ancestor of entity component:default/backstage-janus',
+      'GET',
+      '/api/catalog/entities/by-name/component/default/backstage-janus/ancestry',
+    );
+  });
+
+  test('Should fetch logs for QueriedCatalogEntityFetch event and validate log structure and values', async () => {
+    await uiHelper.clickButton('Create');
+    await validateCatalogLogEvent(
+      'QueriedCatalogEntityFetch',
+      'Queried entity fetch attempt',
+      'GET',
+      '/api/catalog/entities/by-query',
+    );
+  });
+
+  test('Should fetch logs for CatalogLocationCreation event and validate log structure and values', async () => {
+    const template =
+      'https://github.com/RoadieHQ/sample-service/blob/main/demo_template.yaml';
+    await uiHelper.clickButton('Create');
+    await uiHelper.clickButton('Register Existing Component');
+    await catalogImport.analyzeComponent(template);
+
+    await validateCatalogLogEvent(
+      'CatalogLocationCreation',
+      template,
+      'POST',
+      '/api/catalog/locations?dryRun=true',
```

Comment:
are we getting audit logs fro dry runs?  I thought that was disabled because it could result in a ton of unnecessary events.

Reply:
confirmed in Slack that these are still being emitted